naughty-from.sh

```diff
@@ -53,7 +53,7 @@ _arches() {
 _froms() {
 	bashbrew cat --format '
 		{{- range .TagEntries -}}
-			{{- $.DockerFrom . -}}
+			{{- $.DockerFroms . | join "\n" -}}
 			{{- "\n" -}}
```

Comment:
For folks affected by the `DockerFrom` deprecation, in my experience porting my own scripts, this is the most common fix.  (There was one place in the docs we needed to switch to `.ArchLastStageFrom` instead, but it's pretty rare.)